src/requirements/Twitter/TwitterRequirement.tsx

```diff
@@ -17,6 +17,10 @@ import TwitterListLink from "./components/TwitterListLink"
 import TwitterTweetLink from "./components/TwitterTweetLink"
 import TwitterUserLink from "./components/TwitterUserLink"
 
+// todo if it's working well I can change the original pluralize function
```

Comment:
It was used because I used some irregular plural words that was not covered by the original pluralize function, but I created a perfect one :) (not this)

Reply:
I removed it